src/content/release-notes.mdx

```diff
@@ -5,6 +5,32 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.21
+
+7 June 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### New Features
+
+- The platform will now obfuscate variables that are longer than 5 characters  <!-- 8090 -->
+- We've added the ability to [set custom NodeSelectors](self-hosted/helm-configuration.mdx#globals)  <!-- 8093 -->
+
+### Improvements
```

Comment:
Should we include the change we did to remove name validation in ingresses created for cert-manager challenges? It would be something:

`Removed ingress name validation in the mutation webhook for ingress created as part of cert-manager challenges`

Reply:
Oh yes, I'll add that in